meta-lmp-bsp/dynamic-layers/freescale-layer/recipes-bsp/imx-atf/imx-atf_%.bbappend

```diff
@@ -13,7 +13,9 @@ SRC_URI_append_toradex = " \
     file://0001-Revert-Add-NXP-s-SoCs-partition-reboot-support.patch \
 "
 
-do_deploy_append_mx8m() {
-    install -m 0644 ${S}/build-optee/${ATF_PLATFORM}/release/bl31.bin ${DEPLOYDIR}/arm-trusted-firmware.bin
-    install -m 0644 ${S}/build-optee/${ATF_PLATFORM}/release/bl31/bl31.elf ${DEPLOYDIR}/arm-trusted-firmware.elf
+do_deploy_append_mx8() {
+    #install -m 0644 ${S}/build-optee/${ATF_PLATFORM}/release/bl31.bin ${DEPLOYDIR}/arm-trusted-firmware.bin
+    #install -m 0644 ${S}/build-optee/${ATF_PLATFORM}/release/bl31/bl31.elf ${DEPLOYDIR}/arm-trusted-firmware.elf
+    install -m 0644 ${S}/build/${ATF_PLATFORM}/release/bl31/bl31.elf ${DEPLOYDIR}/arm-trusted-firmware.elf
```

Comment:
@mike-scott interesting, that if board doesn't have `optee` MACHINE_FEATURE, the path is different for TF-A
`build-optee` vs `build`
not sure how to address that properly